src/util/collections/stringset.go

```diff
@@ -0,0 +1,76 @@
+package collections
```

Comment:
Don't make this a general util nor a public type unless you also make it threadsafe